File: src/main/java/com/linkhogar/application/house/getById/RoomResponse.java
package com.linkhogar.application.house.getById;

import com.linkhogar.domain.room.enums.RoomStatus;

import java.util.List;

public record RoomResponse(
        String id,
        String name,
        String description,
        Long price,
        Double size,
        Boolean hasPrivateBath,
        String bedType,
        RoomStatus status,
        TenantProfileResponse currentTenant,
        List<String> images
) {}

File: linkhogar-web/src/app/Models/Houses/RoomResponse.ts
import {TenantProfileResponse} from './TenantProfileResponse';

export interface RoomResponse {
  id: string;
  name: string;
  description: string;
  price: number;
  size: number;
  hasPrivateBath: boolean;
  bedType: string;
  status: string;
  currentTenant?: TenantProfileResponse;
  images: string[]; // 👈 El nombre exacto que viene del backend
}
